TabloidCLI/UserInterfaceManagers/TagManager.cs

```diff
@@ -121,7 +121,31 @@ private void Edit()
 
         private void Remove()
         {
-            throw new NotImplementedException();
+            Tag tagToDelete = Choose("Which tag would you like to delete?");
+            if(tagToDelete == null)
+            {
+                return;
+            }
+            Console.WriteLine();
+            Console.WriteLine($"Delete {tagToDelete.Name}? (Y/N)");
+            string response = Console.ReadLine().ToLower();
+            if(response == "y")
+            {
+                _tagRepository.Delete(tagToDelete.Id);
```

Comment:
what happens if there are blogTags associated with this tagId when this delete is attempted?

Reply:
nah, you can go with that for now.